inference-engine/src/mkldnn_plugin/nodes/mkldnn_conv_node.cpp

```diff
@@ -47,7 +48,20 @@ MKLDNNConvolutionNode::MKLDNNConvolutionNode(const std::shared_ptr<ngraph::Node>
         IE_THROW(NotImplemented) << errorMessage;
     }
 
-    isPrimitivesPriorityDefined = op->get_rt_info().count("PrimitivesPriority") != 0;
+    internalBlobDesc.emplace_back([&](primitive_desc_iterator &primitive_desc_it, size_t idx) -> MKLDNNMemoryDesc {
+        return MKLDNNMemoryDesc(primitive_desc_it.weights_desc(0));
+    });
+    internalBlobDesc.emplace_back([&](primitive_desc_iterator &primitive_desc_it, size_t idx) -> MKLDNNMemoryDesc {
+        if (!withBiases)
+            return MKLDNNMemoryDesc();
+        return MKLDNNMemoryDesc(primitive_desc_it.weights_desc(1));
+    });
+
+    if (!implPriorities.empty()) {
+        isPrimitivesPriorityDefined = true;
+        isWino = std::find(implPriorities.begin(), implPriorities.end(), impl_desc_type::jit_avx512_winograd) != implPriorities.end();
+        isWino &= mkldnn::impl::cpu::x64::mayiuse(mkldnn::impl::cpu::x64::avx512_common);
```

Comment:
can we put isWino conditions in one place?

Reply:
done